packages/mui-joy/src/SvgIcon/SvgIcon.test.js

```diff
@@ -129,4 +129,13 @@ describe('<SvgIcon />', () => {
       expect(container.firstChild).to.have.attribute('viewBox', '-4 -4 24 24');
     });
   });
+
+  it('should not override internal ownerState with the ownerState passed to the icon', function test() {
+    if (/jsdom/.test(window.navigator.userAgent)) {
+      this.skip();
+    }
+
+    const { container } = render(<SvgIcon ownerState={{ fontSize: 'sm' }}>{path}</SvgIcon>);
```

Comment:
Is there a real use case to test this one? I ask this because I plan to change the test file to `.tsx` and I am pretty sure that the `SvgIcon` does not support `ownerState` prop.

Reply:
Yes, the icon slot in MUI X Pickers does pass `ownerState` for CSS customization based on ownerState. Also, you can see it in the _before_ and _after_ CodeSandboxes in the description, so we need it.

You can use `// @ts-expect-error` comment.